src/fio/FioEngine.ts

```diff
@@ -939,62 +951,7 @@ export class FioEngine extends CurrencyEngine<FioTools, SafeFioWalletInfo> {
     return res
   }
 
-  // Check all account balance and other relevant info
-  async checkAccountInnerLoop(): Promise<void> {
-    const currencyCode = this.currencyInfo.currencyCode
-
-    // Initialize balance
-    if (
-      typeof this.walletLocalData.totalBalances[currencyCode] === 'undefined'
-    ) {
-      this.updateBalance(currencyCode, '0')
-    }
-
-    // Balance
-    try {
-      const balances = { staked: '0', locked: '0' }
-      let balanceRes: BalanceResponse
-      try {
-        balanceRes = asGetFioBalanceResponse(
-          await this.multicastServers('getFioBalance')
-        )
-      } catch (e: any) {
-        if (e?.json?.message === MAINNET_LOCKS_ERROR) {
-          // This is a fallback query for this chain bug https://fioprotocol.atlassian.net/wiki/spaces/DAO/pages/852688908/2024-02-07+Token+Locking+Issue+on+Unstake
```

Comment:
I noticed that this comment didn't get copied over. Is it somehow irrelevant now?

Reply:
Still relevant, will add back